bios/stephen-bero.md

```diff
@@ -0,0 +1,4 @@
+Stephen Bero
```

Comment:
Hi Stephen

You might add Replace your title with ## Stephen Bero to show a bold title.